### PR TITLE
New function to get current URL of the page

### PR DESCRIPTION
**Pull request by Amit-Kumar-Tiwari-Webkul**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop 
| Description?      | Provide a new function in the Tools class to get the current URL of the page
| Type?             | improvement
| Category?         |  CO 
| BC breaks?        | no
| Deprecations?     |  no
| How to test?      | Indicate how to verify that this change works as expected.
| Fixed ticket?     | Fixes https://github.com/PrestaShop/PrestaShop/discussions/33092
| Sponsor company   | Your company or customer's name goes here (if applicable).

### CODE REVIEW

File: classes/Tools.php [hunk 1]
```diff
@@ -3858,6 +3858,16 @@ public static function arrayInsertElementAfterKey(array $array, string $targetKe
 
         return $array;
     }
+
+    /**
+     * Get the current URL of the page
+     *
+     * @return string
+     */
+    public static function getCurrentPageURL()
```
Comment (kpodemski) [inline classes/Tools.php]:
```suggestion
    public static function getCurrentPageURL(): string
```

**Review by kpodemski — CHANGES_REQUESTED** [review]
We use strict types, let's use return type there.

**Comment by PrestaEdit** [comment]
Hi,

What's the difference/needs to get this in addition of `getCurrentUrl()` ?

**Comment by kpodemski** [comment]
Hi @Amit-Kumar-Tiwari-Webkul 

@PrestaEdit is right, we do have such a method available already. Could you check?

PS: Sorry for this previous review, I was showing somebody how to do the review without getting familiar better with your PR, sorry 🙏🏻

**Comment by Amit-Kumar-Tiwari-Webkul** [comment]
Thanks for the information.
It is already available on the new PS version 8.1

**Pull request closed without merging**